src/wovnio/wovnphp/Headers.php

```diff
@@ -309,11 +309,15 @@ public function removeLang($uri, $lang = null)
         }
     }
 
-    public function getDocumentURI()
+    public function getDocumentURI($withQuery = false)
     {
         $url = $this->env['REQUEST_URI'];
         $url_arr = parse_url($url);
 
+        if ($withQuery) {
+            return $url;
```

Comment:
The following code is skipped.
```
return $this->removeLang($uri, $this->lang());
```

Reply:
Say no to boolean function parameters. I especially don't agree with using it just as a way to skip a good chunk of the method (that url parsing becomes useless).

I've been thinking of removing the `getDocumentURI` function entirely since it doesn't seem to be doing much. So I wish we weren't adding to it.

Reply:
By the way, since I am removing the query feature (which seems to be what is blocking you here). There might not even be a need for this change?

Reply:
I don't think this is related to the query feature you are removing tho. It's just that in the past for some reason when pulling the URI to determine if it should be ignored, we chose to get rid of the queries in it.

Reply:
Yes, in my PR I also removed this code because I deemed it useless 😛 Try merging with master and see if you actually need to make any changes.

Reply:
Ahh I see. Yep should work fine.